cairo/ethereum/cancun/vm/instructions/log.cairo

```diff
@@ -0,0 +1,445 @@
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
+
+from ethereum.cancun.vm.stack import pop
+from ethereum.cancun.vm import Evm, EvmImpl
+from ethereum.cancun.vm.exceptions import ExceptionalHalt, WriteInStaticContext, OutOfGasError
+from ethereum.cancun.vm.memory import memory_read_bytes, expand_by
+from ethereum.cancun.vm.gas import calculate_gas_extend_memory, charge_gas, GasConstants
+from ethereum.utils.numeric import U256_to_be_bytes
+from ethereum_types.numeric import U256, U256Struct, Uint
+from ethereum_types.bytes import Bytes, Bytes32, Bytes32Struct, TupleBytes32, TupleBytes32Struct
+from ethereum_types.others import (
+    ListTupleU256U256,
+    ListTupleU256U256Struct,
+    TupleU256U256,
+    TupleU256U256Struct,
+)
+from ethereum.cancun.blocks import Log, LogStruct, TupleLog, TupleLogStruct
+
+// @notice LOG0 instruction - append log record with no topics
+func log0{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, evm: Evm}() -> ExceptionalHalt* {
```

Comment:
where is the bitwise used?

Reply:
for U256_to_be_bytes since uint256_reverse_endian uses it